webpack.config.js

```diff
@@ -271,7 +274,8 @@ module.exports = async (env, argv) => {
       headers: {
         "Access-Control-Allow-Origin": "*"
       },
-      inline: !env.bundleAnalyzer,
+      hot: false,
```

Comment:
I generally do like having hot reloading most of the time. Maybe expose as an env var? I also dont really mind just editing my webpack config locally though, so not a very strong opinion.

Reply:
Cool, added it.